plugins/channelrx/demodnfm/nfmdemodsink.cpp

```diff
@@ -174,9 +176,11 @@ void NFMDemodSink::processOneSample(Complex &ci)
         if (!m_settings.m_audioMute && (m_settings.m_ctcssOn && m_ctcssIndexSelected == ctcssIndex || m_ctcssIndexSelected == 0))
         {
             Real audioSample = m_squelchDelayLine.readBack(m_squelchGate);
+            Q_ASSERT(audioSample >= -1.0f && audioSample <= 1.0f);
```

Comment:
Please do not add code that will make the whole app crash because of a particular audio sample.

Reply:
This will trigger only in debug build. And it is good reminder that we should work with values [-1, 1] at this stage. But if you think it is too much  I will remove it.

Reply:
Even in debug mode you would not like to be polluted by a problem that you are not currently investigating moreover if I'm right there is a clamping a bit further down.